lldb/tools/lldb-dap/lldb-dap.cpp

```diff
@@ -5046,30 +5026,50 @@ int main(int argc, char *argv[]) {
     pause();
   }
 #endif
+
+  // Initialize LLDB first before we do anything.
+  lldb::SBDebugger::Initialize();
+
+  // Terminate the debugger before the C++ destructor chain kicks in.
+  auto terminate_debugger =
+      llvm::make_scope_exit([] { lldb::SBDebugger::Terminate(); });
+
+  DAP dap;
+  dap.debug_adaptor_path = program_path.str().str();
+  dap.repl_mode = defaultReplMode;
```

Comment:
can you add the debug_adaptor_path and the repl_mode in the constrsuctor of the Dap class?

Reply:
Done, moved those both to the constructor.